api/resources_portal/views/import_sra_utils.py

```diff
@@ -18,6 +18,28 @@ class UnsupportedDataTypeError(Exception):
     pass
 
 
+def _get_number_of_samples(srs_string):
+    """
+    SRS accession codes are provided in the following format:
+    "SRS000001, SRS000002, SRS000003-SRS000008"
+    You can count the non-consecutive elements, but the ranges must be parsed.
+    """
+
+    num_samples = 0
+
+    srs_list = srs_string.split(",")
+
+    for srs in srs_list:
+        srs = srs.replace("SRS", "")
```

Comment:
Wait, we want number of runs, not samples (I mentioned that in https://github.com/AlexsLemonade/resources-portal/pull/248#discussion_r457632161). I can understand why this is confusing, because like, why wouldn't we want to count the sample accessions for number of samples? However, an `SRS` accession is generated for each chunk of organism that is collected, which may then be run through the machine multiple times. Each time it gets run through the machine, an `SRR` accession is generated. In refinebio we've used `samples` to refer to runs, since each one is a new column of gene expression data.

I think for this project the same holds true, since if you download a dataset with 4 samples and then see 16 columns it'd probably be more surprising than finding out that a dataset claiming to have 16 samples was actually creating using only 4 chunks of organism. Can you confirm this @dvenprasad ?

Reply:
Yep. that sounds reasonable to me.